test/lib/jdk/test/lib/threaddump/ThreadDump.java

```diff
@@ -296,6 +296,16 @@ public String parkBlocker() {
             return getStringProperty("parkBlocker", "object");
         }
 
+        /**
+         * Returns the owner of the parkBlocker if the parkBlocker is an AQS.
```

Comment:
Can you change "AQS" to AbstractOwnableSynchronizer here? It could be changed to AOS too but few will know what it means.